src/ts/proposal.ts

```diff
@@ -107,15 +120,70 @@ export async function generateProposal(
       safeDeploymentAddresses,
       ethers,
     );
+  const cowTokenContract = await ethers.getContractAt(
+    "CowProtocolToken",
+    cowToken,
+  );
+  const multiTokenMediator = await ethers.getContractAt(
+    "IOmnibridge",
+    settings.bridge.multiTokenMediatorETH,
+  );
+  const approvalOmniBridgeTx = {
+    to: cowToken,
+    value: 0,
+    data: cowTokenContract.interface.encodeFunctionData("approve", [
+      settings.bridge.multiTokenMediatorETH,
+      settings.bridge.amountToRelay,
+    ]),
+    operation: 0,
+  };
+
+  const relayTestFundsToOmniBridgeTx = {
+    to: settings.bridge.multiTokenMediatorETH,
+    value: 0,
+    data: multiTokenMediator.interface.encodeFunctionData("relayTokens", [
+      cowToken,
+      cowDao,
+      settings.bridge.amountToRelay,
+    ]),
+    operation: 0,
+  };
+
+  const transferCowTokenToCowDao = {
+    to: cowToken,
+    value: 0,
+    data: cowTokenContract.interface.encodeFunctionData("transfer", [
+      cowDao,
+      totalSupply.sub(settings.bridge.amountToRelay),
+    ]),
+    operation: 0,
+  };
```

Comment:
I think it would make sense to capture this in a single function `bridgeTokensToGnosisChain` that returns an array of the transactions that need to be executed.